articles/sentinel/customer-managed-keys.md

```diff
@@ -19,18 +19,32 @@ ms.author: yelevin
 ---
 # Set up Azure Sentinel customer-managed key
 
-This article provides background information and steps to configure a customer-managed key (CMK) for Azure Sentinel. CMK enables all data saved or sent to Azure Sentinel to be encrypted in all relevant storage resources with an Azure Key Vault key created or owned by you.
+This article provides background information and steps to configure a customer-managed key (CMK) for Azure Sentinel. CMK enables all data stored in Azure Sentinel to be encrypted in all relevant storage resources with an Azure Key Vault key created and owned by you.
 
 > [!NOTE]
-> - The Azure Sentinel CMK capability is provided only to **new customers**.
+> - The Azure Sentinel CMK capability is provided only to **new workspaces**.
 >
-> - Access to this capability is controlled by Azure feature registration. You can request access by contacting azuresentinelCMK@microsoft.com. Pending requests will be approved according to the available capacity.
+> - Onboarding a CMK workspace to Sentinel is supported only via REST API and not via the Azure Portal. Azure Resource Manager (ARM) templates are currently not supported for CMK onboarding.
 >
-> - The CMK capability is only available to customers sending 1TB per day or more. You will receive information about additional pricing when you apply to Microsoft to provision CMK on your Azure subscription. Learn more about [Log Analytics pricing](../azure-monitor/logs/manage-cost-storage.md#log-analytics-dedicated-clusters).
+> - Once a workspace is onboarded as a CMK workspace, setting it as a non-CMK workspace by de-linking it from the dedicated Log Analytics CMK cluster, or setting the dedicated Log Analytics cluster as non-CMK is not supported and might lead to undefined and undesired behavior.
+>
+> - Once a workspace was onboarded to Azure Sentinel as a non-CMK workspace, linking it to a CMK Log Analytics cluster, or setting the Log Analytic cluster its currently linked to, as a CMK cluster, is not supported, and will lead to its Azure Sentinel data being encrypted by a Microsoft Managed Key instead of the Customer Managed Key.
+>
+> - Currently, only System Assigned Identities are supported with Azure Sentinel CMK. Therefore, the dedicated Log Analytics cluster's identity should be of System Assigned type. It is recommended that it will be the identity automatically assigned to the Log Analytics cluster upon its creation.
```

Comment:
@ehudklein  can we make this line more positive? 
like we support Assigned Identities with Azure Sentinel CMK